.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: java
+jdk:
+  - openjdk11
+script:
+  - "./gradlew clean build"
+
+before_deploy: "mv build/libs/gocd-google-chat-build-notifier-uber.jar build/libs/gocd-google-chat-build-notifier-plugin.jar"
```

Comment:
this is not required. `./gradlew clean build` puts a jar under `build/libs`

Reply:
Done.